src/lib/export/exporters/tabby.ts

```diff
@@ -0,0 +1,36 @@
+import { toQueryString, type Recipe } from "$lib/ingredients";
+import { prepare } from "$lib/cereals";
+
+export function toTabby(recipe: Recipe): string {
+  const queryString = toQueryString(recipe);
+  const cereals = prepare(recipe);
+
+  return `# Copy the configuration below and add it under
+# terminal.customColorSchemes in the appropriate config file:
+# Linux: ~/.config/tabby/config.yaml
+# macOS: ~/Library/Application Support/tabby/config.yaml
+# Windows: ~\\AppData\\Roaming\\tabby\\config.yaml
+# Open Tabby's Settings > Color scheme and select the 'Root Loops' theme.
+
+    - name: Root Loops # via https://rootloops.sh?${queryString}
```

Comment:
Just a small tweak: Can we move the `# via rootloops.sh?...` comment to the comment block above? It just looks a bit cleaner that way.

Reply:
Sorry for the delay on getting back to this. So I *can* move it, but in my testing, Tabby ends up eating comments that are on their own line when edits are made using the Settings UI.

So your intention for these comments is to have them be documentation that lives in the config file, keeping it on the same line is the only feasible way for Tabby. If it's not, happy to move.